package.json

```diff
@@ -223,7 +223,7 @@
     "test:system": "PLUGINS_PATH=\"${npm_config_externalplugins}\" ./scripts/run-system-tests",
     "test:validate": "PLUGINS_PATH=\"${npm_config_externalplugins}\" ./scripts/validate-tests",
     "test:watch": "npm test -- --watch",
-    "test": "lingui compile && NODE_PATH=node_modules jest --config=jest.config.js --no-cache",
+    "test": "lingui compile && NODE_PATH=node_modules jest --config=jest.config.js --no-cache --forceExit --detectOpenHandles",
```

Comment:
`--detectOpenHandles` seems like more of a debug option. Does it reveal anything about jest not exiting?

Reply:
Using `--forceExit` without it jest will print a message suggesting you use it, so I added it. Assuming it will give some info if jest has to forceExit.